### DISCUSSION TITLE
How can I add NoFollow to all external links?

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#283] [Ask a question] [answered]
Hello Developers,

how can I add the noFollow attribute to all external links or add noFollow for the entire web page? Internal links, of course, only for external links.

Thanks in advance.

**Comment by schulle4u** [comment, marked as answer]
Yellow supports the [Markdown Extra](https://michelf.ca/projects/php-markdown/extra/) implementation by Michel Fortin, which also provides extra attributes for headings, links and other elements. Here is an example to add `rel=nofollow` to a link. 

`[Datenstrom](https://datenstrom.se){rel=nofollow}` 

Adding this automatically to all external links however would require some kind of plugin I think. 

HTH
Steffen

**Comment by ghost** [comment]
Ok, well, thanks for the info and the tips. I need not always noFollow, but if I link my own web pages from the Portoflio, I would like the Google and Co. the noFollow attribute.

I will use it now. I can now close the issue.
